example/lib/main.dart

```diff
@@ -80,78 +74,59 @@ class ExampleApp extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
```

Comment:
What has changed here and why?

Reply:
The single screen app has been split into two screens, with a view to adding more in the future.

Currently it's just the "home" and "native crashes" screens, but in ErrorBoundary should also have a dedicated screen.

The main idea was declutter the home screen, while also showing the navigation breadcrumbs in action. Happy to revert if thats preferred.